floodviz/thumbnail/thumbnail.js

```diff
@@ -82,7 +83,7 @@ function convert(figure, window, css_path, filename) {
 		svg_string = inject_style(style_default, null, svg, window);
 	}
 	// Takes care of canvas conversion and encodes base64
-	svg2png(svg_string)
+	svg2png(svg_string, {height: 300, width: 560})
```

Comment:
I did however, take your suggestion and declare vars at the top for height and width instead of using the actual integers.